test/integration/acceptance/add_asset_qty_test.cpp

```diff
@@ -174,18 +123,17 @@ TEST_F(AddAssetQuantity, Uint256DestOverflow) {
 
 /**
  * @given some user with all required permissions
- * @when execute tx with AddAssetQuantity command with inexistent account
+ * @when execute tx with AddAssetQuantity command with nonexistent account
  * @then there is an empty proposal
  */
-TEST_F(AddAssetQuantity, InexistentAccount) {
-  const std::string &inexistent = "inexist@test"s;
-  IntegrationTestFramework()
+TEST_F(AddAssetQuantity, NonexistentAccount) {
+  const std::string &nonexistent = "inexist@test"s;
```

Comment:
Not really sure this convoluted form grants us any benefits. IMO it better to write
`std::string nonexistent = "blabla"`. We still perform one construction, but a reader of the code no longer has to be aware of things such as lifetime extensions etc.

Reply:
Regarding `operator""s`, it has a profit of compile-time introspection of string length, which does not occur in `std::string("str")`, so we can experiment with it in methods which take `std::string` object, but we pass `const char*` in it.